src/SFA.DAS.ApplyService.InternalApi/Startup.cs

```diff
@@ -56,7 +56,7 @@ public Startup(IConfiguration configuration, IHostingEnvironment env)
         public IServiceProvider ConfigureServices(IServiceCollection services)
         {
 
-            IdentityModelEventSource.ShowPII = true;
+            IdentityModelEventSource.ShowPII = false;
```

Comment:
You can actually remove this line as it defaults to false.

Reply:
Yep. I know and thought about that.  Can do.